tools/rna_tools/rnacode/rnacode.xml

```diff
@@ -1,17 +1,26 @@
-<tool id="rbc_rnacode" name="RNAcode" version="0.3.0">
+<tool id="rbc_rnacode" name="RNAcode" version="0.3.1">
     <description>Analyze the protein coding potential in MSA.</description>
     <requirements>
         <requirement type="package" version="0.3">rnacode</requirement>
     </requirements>
-    <stdio>
-        <exit_code range="1:"  level="fatal" description="Error occurred. Please check Tool Standard Error"/>
-        <exit_code range=":-1" level="fatal" description="Error occurred. Please check Tool Standard Error"/>
-    </stdio>
     <version_command>RNAcode --version</version_command>
-    <command>
+    <command detect_errors="exit_code">
     <![CDATA[
-        RNAcode
-        
+        #if $cond_breakmaf.select_breakmaf == 'breakmaf'
+            $__tool_directory__/breakMAF.pl
```

Comment:
This one should be quoted.

Reply:
@bgruening if I quote $cond_breakmaf.select_breakmaf then it does not work (i.e. the else path is used all the time). I guess you meant to quote something else?

Reply:
@bernt-matthias You need to quote `$__tool_directory__/breakMAF.pl`, i.e. `'$__tool_directory__/breakMAF.pl'` , since `$__tool_directory__` may contain spaces.